hyperjumptech/monika
# Feat/presistent logs

# PR Review

## What does this PR fixes

1. This PR implementes #23 
2. Historical logs are now stored in an sqlite db

## How was it implemented

1. Installing sqlite3
2. Created a table called history
3. Probe results are inserted into the logs

## How to test

1. run `bin/run`  should work. You should get statuscode sensitive coloring
2. run `bin/run -l` should dump the logs
3. run `bin/run --flush` should delete the logs. Only a "Y" character should be read as a yes.
